### PR TITLE
Misc PAT cleanup including FQTN fix

### PR DESCRIPTION
**Pull request by dekatzenel**
### Background

This PR has two main components:

1. Fix for FQTN check not supporting all snowflake syntax
2. Misc PAT improvements found during both repo analysis and usage

### Changes

* FQTN check now uses sqlfluff instead of sql_metadata plus improved testing
* Print test errors during non-test commands
* replace unicode characters from backend with actual characters
* rename cmd dir to command so that the pycharm debugger works (got put too early in the classpath)
* minor refactoring

### Testing

* Added new unit tests
* Ensured existing tests passed
* Verified licenses on new packages

### CODE REVIEW

**Comment by darwayne** [comment]
This is awesome. Thanks for getting this out so quickly. Tomorrow morning I'll run some manual testing to double check we are good

**Review by maxrichie5 — APPROVED** [review]
Thanks! Just one question

File: panther_analysis_tool/util.py [hunk 1]
```diff
@@ -165,3 +166,10 @@ def deep_get(obj: Dict, path: List[str], default: Any = None) -> Any:
 def to_list(listish: Any) -> List:
     """Make a single instance a list or keep a list a list."""
     return listish if isinstance(listish, list) else [listish]
+
+
+def convert_unicode(obj: Any) -> str:
+    """Swap unicode 4 byte strings with arbitrary numbers of leading slashes with the actual character
+    e.g. \\\\u003c => <"""
+    string_to_convert = str(obj)
+    return re.sub(r"\\*\\u([0-9a-f]{4})", lambda m: chr(int(m.group(1), 16)), string_to_convert)
```
[the remark this answers is not in the document]
Comment (dekatzenel) [reply panther_analysis_tool/util.py:175]: The issue details returned from the BE are utf-encoded, so they'll say things like (example, not a quote) `"value must be \\u003c 100"` and it's super annoying

[the remark this answers is not in the document]
Comment (maxrichie5) [reply panther_analysis_tool/util.py:175]: I do like that you are fixing that. But maybe this ought to be fixed on the backend instead of here?

**Review by darwayne — COMMENTED** [review]
Verified this works as expected on panther-analysis. CI & License issues need to be addressed though

File: panther_analysis_tool/util.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dekatzenel) [reply panther_analysis_tool/util.py:175]: It's not obvious to me that that's true given that the website is handling this correctly

[the remark this answers is not in the document]
Comment (maxrichie5) [reply panther_analysis_tool/util.py:175]: As in something between PAT and the backend is causing this?

**Comment by darwayne** [comment]
re: failing ci check ... I think the make integration script needs to be updated .. I suspect that is pulling down the current release vs current PR

File: panther_analysis_tool/util.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dekatzenel) [reply panther_analysis_tool/util.py:175]: More like something between the BE and the web may be requiring this encoding, so PAT has to deal with it. That's pure speculation, though

**Review by nhakmiller — APPROVED** [review]

**Review by darwayne — APPROVED** [review]

File: panther_analysis_tool/util.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (darwayne) [reply panther_analysis_tool/util.py:175]: I think what's here is fine. We can make a follow up task to see if there's a better approach to solving this prob.

**Pull request merged**